install.py

```diff
@@ -0,0 +1,150 @@
+    os.chdir("..")
+    return iwi_dir_name
+
+def input_datastore_properties():
+    properties = {}
+    print("Datastore Service Configuration Properties")
+    properties[oauth_storage_service_token_url]=input(oauth_storage_service_token_url + "=")
+    properties[oauth_storage_service_client_id]=input(oauth_storage_service_client_id + "=")
+    properties[oauth_storage_service_client_secret]=input(oauth_storage_service_client_secret + "=")
+    return properties
+
+def input_service_host_addresses():
+    addresses = {}
+    print("Service Host Addresses")
+    print("Requires http prefix 'http://")
+    addresses[storage_service_host_url]=input(storage_service_host_url + "=")
+    addresses[iguassu_service_host_url]=input(iguassu_service_host_url + "=")
+    addresses[arrebol_service_host_url]=input(arrebol_service_host_url + "=")
+    addresses[web_ui_host_url]=input(web_ui_host_url + "=")
+    print('\n')
+    return addresses
+
+def get_monitors_periods():
+    periods = {}
+    periods[job_state_monitor_period]=5000
+    periods[session_monitor_period]=3600000
+    periods[job_submission_monitor_period]=5000
+    return periods
+
+def write_conf_file(confs, path, file_name):
+    os.chdir(path)
+    f = open(file_name, "w+")
+    for key, value in confs.items():
+        f.write(str(key)+"="+str(value)+"\n")
+    f.close()
+    os.chdir("..")
+
+def get_web_ui_conf(confs):
+    ui_conf = {}
+    ui_conf[VUE_APP_EXTERNAL_LINK_DOCS] = "https://nicedoc.io/ufcg-lsd/iguassu"
+    ui_conf[VUE_APP_EXTERNAL_LINK_GITHUB_REPO] = "https://github.com/ufcg-lsd/iguassu"
+    ui_conf[VUE_APP_EXTERNAL_LINK_ABOUT] = "http://cloudlab-brasil.rnp.br/iguassu"
+
+    ui_conf[VUE_APP_OWNCLOUD_SERVER_HOST] = confs[storage_service_host_url]
+    ui_conf[VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID] = confs[oauth_storage_service_client_id]
+    ui_conf[VUE_APP_OWNCLOUD_OAUTH_SECRET] = confs[oauth_storage_service_client_secret]
+    ui_conf[VUE_APP_OWNCLOUD_OAUTH_REDIRECT_URI] = confs[web_ui_host_url] + "/auth"
+
+    ui_conf[VUE_APP_IGUASSU_API] = confs[iguassu_service_host_url]
+
+    return ui_conf
+
+def write_web_ui_conf(confs, web_ui_dir_name):
+    ui_confs = get_web_ui_conf(confs)
+    write_conf_file(ui_confs, web_ui_dir_name, ".env")
+
+def write_iguassu_conf(confs, iguassu_dir_name):
+    confs.pop(web_ui_host_url, None)
+    write_conf_file(confs, iguassu_dir_name, "iguassu.conf")
+
+def main():
+    print("---> Installing Iguassu System Service\n")
+    print("Configure the following settings...\n")
+    ds_properties = input_datastore_properties()
+    host_addresses = input_service_host_addresses()
+    monitors_periods = get_monitors_periods()
+    confs = {**ds_properties, **host_addresses, **monitors_periods}
+
+    iguassu_dir_name = install_iguassu()
+    arrebol_dir_name = install_arrebol()
+    web_ui_dir_name = install_web_ui()
+
+    write_web_ui_conf(confs, web_ui_dir_name)
+    write_iguassu_conf(confs, iguassu_dir_name)
+
```

Comment:
It a good idea clear all the temporary both directories and zip files at the finish of the script or in any case of error, maybe is the case of use `try-finally` statements.